fast_flights/parser.py

```
@@ -17,47 +14,104 @@
 
+def parse(html: str) -> MetaList:
+    # Try multiple ways to find the data
+    
+    # 1. Classic script tag with ID or class
+    parser = LexborHTMLParser(html)
+    for selector in [r"script.ds\:1", r"script#ds\:1", r"script[data-id='ds:1']"]:
+        script = parser.css_first(selector)
+        if script and "data:" in script.text():
+            try:
+                return parse_js(script.text())
+            except:
```

Ruff (0.16.0)

[error] Do not use bare `except`

(E722)